portal-kernel/test/unit/com/liferay/portal/kernel/module/service/SnapshotTest.java

```diff
@@ -256,6 +256,34 @@ public void testInvalidFilter() throws InvalidSyntaxException {
 		serviceRegistration.unregister();
 	}
 
+	@Test
+	public void testNotFoundBundle() {
+		try {
+			Mockito.when(
+				FrameworkUtil.getBundle(Mockito.any())
+			).thenReturn(
+				null
+			);
+
+			Snapshot<TestService<String>> snapshot = new Snapshot<>(
+				SnapshotTest.class, Snapshot.cast(TestService.class));
+
+			snapshot.get();
+
+			Assert.fail();
+		}
+		catch (Exception exception) {
+			Assert.assertSame(NullPointerException.class, exception.getClass());
+		}
+		finally {
+			Mockito.when(
+				FrameworkUtil.getBundle(Mockito.any())
```

Comment:
@W2jx Is it possible to avoid swapping this mock, and use some specific parameter to map to different cases instead?